python/llm/src/ipex_llm/transformers/pipeline_parallel.py

```diff
@@ -106,6 +107,29 @@ def init_pipeline_parallel():
     dist.init_process_group('ccl')
 
 
+def _check_quantize_kv_cache(model, idx, batch_size) -> bool:
```

Comment:
Does this function return a bool value?

Reply:
> Does this function return a bool value?

Fix it. No need to return bool value here :)